src/Microsoft.IdentityModel.Clients.ActiveDirectory/Microsoft.IdentityModel.Clients.ActiveDirectory.csproj

```diff
@@ -135,6 +135,12 @@
     <AndroidResource Include="Resources\**\*.axml" />
     <AndroidResource Include="Resources\**\*.xml" />
   </ItemGroup>
+  <PropertyGroup>
+    <CodeAnalysisRuleSet>..\..\ConfigureAwait.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" />
+  </ItemGroup>
```

Comment:
pls. ensure that all projects follow this ruleset, including test apps, unit test projects, etc.

Reply:
It turns out that the suppression file is automatically included within this project. No reference is needed. same is true with a number of other projects. Only a few projects needed to have this included.